hbase-client/src/main/java/org/apache/hadoop/hbase/ServerMetrics.java

```diff
@@ -106,4 +106,6 @@ default String getVersion() {
   @Nullable
   List<ServerTask> getTasks();
 
+  /** Returns the region prefetch information for the regions hosted on this server */
+  Map<String, Integer> getRegionPrefetchInfo();
```

Comment:
Nit: Mind explain in the comment what is the Integer value representing here? Is it the total region size, or amount of the region size cached? If the former, is that absolute or round percentage?

Reply:
Updated the comment to add information about about the integer value in the regionPrefetchInfo.